### PR TITLE
Removed Ckey restrictions from certain genemod options

### PR DESCRIPTION
**Pull request by 990xan**
Removed Ckey restrictions from certain genemod options, if I missed any let me know

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
Remove the accompanying names, too.
If you'd also be so kind as to remove the tailmaw, we'd be golden.

**Comment by 990xan** [comment]
alright, I'll do that

**Comment by 990xan** [comment]
I think I got them all

**Comment by ProbablyCarl** [comment]
Mad lad. 
The moment Travis is done, I'll merge this.

**Comment by 990xan** [comment]
lit

File: modular_mithra/code/modules/mithra/sprite_accessories_vr.dm [hunk 1]
```diff
@@ -776,13 +763,6 @@
 	color_blend_mode = ICON_MULTIPLY
 	do_colouration = 1
 
-/datum/sprite_accessory/tail/tailmaw
```
Comment (chaoko99) [inline modular_mithra/code/modules/mithra/sprite_accessories_vr.dm:779]: Undocumented removal?

**Comment by ProbablyCarl** [comment]
> Remove the accompanying names, too.
> If you'd also be so kind as to remove the tailmaw, we'd be golden.

@chaoko99

**Pull request merged**